app/assets/javascripts/govuk_publishing_components/analytics/auto-scroll-tracker.js

```diff
@@ -31,9 +31,21 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       return
     }
 
+    this.trackType = this.$module.getAttribute('data-track-type')
+    var trackHeadings = this.$module.getAttribute('data-track-headings')
+    if (trackHeadings) {
+      try {
+        // Valid JSON doesn't use '', but we can use replace to be forgiving
+        this.config.trackHeadings = JSON.parse(trackHeadings.replace(/'/g, '"'))
+      } catch {
```

Comment:
Should we do anything with the error? it seems like it'd be easy for no-one to know there is a problem here - perhaps console.error?

Reply:
Good idea, I've put a console.error in.

Reply:
Super. I'd suggest including the error message:

```
} catch (e) {
  console.error('Scroll tracker configuration error: ' + e.message, window.location)
  // ...
}
```

Reply:
Updated as suggested, thanks.